### DISCUSSION TITLE
K8S Helm: Use S3 as objectStrore does not work

### DISCUSSION THREAD

**Discussion by vaibhav-goyal** [Budibase/budibase#8893] [Help]
I cannot get S3 as objectStore to work. 
App gets created or imported but screen preview always fails. 

Logs from proxy service show that it is still proxies requests to a non-existent minio service, it should proxy them to S3. 

Let me know if anyone is able to get this work or has any ideas.

**Comment by melohagan** [comment]
Hey @vaibhav-goyal 

I believe this is an open issue: https://github.com/Budibase/budibase/issues/5636
